<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="1" hostname="DESKTOP-GPMOM1I" name="com.appium.keyword.android.AndroidParallelTest" tests="2" failures="0" timestamp="02 Jun 2018 00:42:17 GMT" time="5.967" errors="1">
  <testcase name="testLogin" time="0.000" classname="com.appium.keyword.android.AndroidParallelTest">
    <skipped/>
  </testcase> <!-- testLogin -->
  <testcase name="beforeTest" time="5.967" classname="com.appium.keyword.android.AndroidParallelTest">
    <error type="io.appium.java_client.service.local.InvalidServerInstanceException" message="Invalid server instance exception has occured: The invalid appium node F:\usr\local\lib\node_modules\appium\build\lib\main.js has been defined">
      <![CDATA[io.appium.java_client.service.local.InvalidServerInstanceException: Invalid server instance exception has occured: The invalid appium node F:\usr\local\lib\node_modules\appium\build\lib\main.js has been defined
at io.appium.java_client.service.local.AppiumServiceBuilder.validateNodeStructure(AppiumServiceBuilder.java:102)
at io.appium.java_client.service.local.AppiumServiceBuilder.checkAppiumJS(AppiumServiceBuilder.java:294)
at io.appium.java_client.service.local.AppiumServiceBuilder.createArgs(AppiumServiceBuilder.java:389)
at org.openqa.selenium.remote.service.DriverService$Builder.build(DriverService.java:332)
at com.appium.service.AppiumManager.startAppiumServerForAndroid(AppiumManager.java:75)
at com.appium.locator.InitLocator.getAndroidLocator(InitLocator.java:105)
at com.appium.keyword.android.AndroidParallelTest.beforeTest(AndroidParallelTest.java:32)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:455)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
at org.testng.TestRunner.beforeRun(TestRunner.java:529)
at org.testng.TestRunner.run(TestRunner.java:497)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
at java.lang.Thread.run(Thread.java:748)
Caused by: java.io.IOException: The node F:\usr\local\lib\node_modules\appium\build\lib\main.jsdoesn't exist
at io.appium.java_client.service.local.AppiumServiceBuilder.validateNodeStructure(AppiumServiceBuilder.java:104)
... 25 more
]]>
    </error>
  </testcase> <!-- beforeTest -->
</testsuite> <!-- com.appium.keyword.android.AndroidParallelTest -->
